packages/application-to-register-webapp/src/routes/__tests__/land/legal-agreement-start-date-spec.js

```diff
@@ -0,0 +1,126 @@
+import { submitGetRequest, submitPostRequest } from '../helpers/server.js'
+import constants from '../../../utils/constants'
+
+const url = '/land/legal-agreement-start-date'
+
+describe(url, () => {
+  describe('GET', () => {
+    it(`should render the ${url.substring(1)} view`, async () => {
+      const response = await submitGetRequest({ url })
+      expect(response.statusCode).toBe(200)
+    })
+  })
+
+  describe('POST', () => {
+    let postOptions
+    beforeEach(() => {
+      postOptions = {
+        url,
+        payload: {}
+      }
+    })
+
+    it('should add a legal agreement start date', async () => {
+      postOptions.payload['legalAgreementStartDate-day'] = '01'
+      postOptions.payload['legalAgreementStartDate-month'] = '01'
+      postOptions.payload['legalAgreementStartDate-year'] = '1971'
```

Comment:
Although, not part of the acceptance criteria I question the service accepting start dates for legal agreements (and other documents) before dates specified by legislation and/or policy. I think this needs confirmation with the product owner.

Reply:
I will bring this up with Shawn